packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -68,7 +68,12 @@ import invariant from 'shared/invariant';
 import warningWithoutStack from 'shared/warningWithoutStack';
 
 import ReactFiberInstrumentation from './ReactFiberInstrumentation';
-import * as ReactCurrentFiber from './ReactCurrentFiber';
+import {
+  getStackByFiberInDevAndProd,
+  phase as ReactCurrentFiberPhase,
```

Comment:
I don't see a pattern in what you use an uppercase first letter for (e.g. `ReactCurrentFiberPhase`) and what stays lower case (e.g. `getInstance`)

Reply:
Either if a) there's more than one with the same name, b) I'm scared of overlap from a local variable or something.

But I think we should probably rename these on a case-by-case basis to something designed for being used with named imports.